commcare-cloud-bootstrap/environment/private.yml.j2

```diff
@@ -1,35 +1,33 @@
 secrets:
-  ECRYPTFS_PASSWORD: 'xxBvYiMZ3YVIVkFR'
+  ECRYPTFS_PASSWORD: '{{ generate_uuid() }}'
   KSPLICE_ACTIVATION_KEY:
   DATADOG_API_KEY: null
   DATADOG_APP_KEY: null
   AMQP_USER: "{{ deploy_env }}_worker"
```

Comment:
this will get rendered as `"_worker"` unless you escape the `{{` or you pass `deploy_env` into the jinja context

Reply:
Didn't notice that, I think it currently doesn't work in master either:
https://github.com/dimagi/commcare-cloud/pull/2040/commits/3a10da86b133453d9a2353d33d332fbc38ff604a

Reply:
My reading of the code here is that it _did_ use to work on master, because before this PR, `private.yml` was copied directly into the new env, and then only later during execution, `ansible`/`ansible-playbook` would plug in the value for `{{ deploy_env }}` here. When you change a text file to a .j2 file, you have to start by escaping all the `{{`s, i.e. you have to replace  `{{` with `{{ '{{' }}`.

That said, I think the solution you landed on will totally work—it's just plugging in the value now so it's hardcoded in the resultant `vault.yml` file inside the env—and is more readable. It results in something different landing in the env `vault.yml` file, but I'd say the change is an improvement.

Reply:
got it